docs/source/overview.rst

```diff
@@ -3,56 +3,66 @@
-
-CuPy additionally supports a subset of SciPy features:
-
-- `Sparse matrices <https://docs.scipy.org/doc/scipy/reference/sparse.html>`_ and `sparse linear algebra <https://docs.scipy.org/doc/scipy/reference/sparse.linalg.html>`_, powered by `cuSPARSE <https://developer.nvidia.com/cusparse>`_.
-- `Multi-dimensional image processing <https://docs.scipy.org/doc/scipy/reference/ndimage.html>`_
-- `Signal processing <https://docs.scipy.org/doc/scipy/reference/signal.html>`_
-- `Fast Fourier transform <https://docs.scipy.org/doc/scipy/reference/fft.html>`__ (FFT)
-- `Linear algebra functions <https://docs.scipy.org/doc/scipy/reference/linalg.html>`__
-- `Special functions <https://docs.scipy.org/doc/scipy/reference/special.html>`_
-- `Statistical functions <https://docs.scipy.org/doc/scipy/reference/stats.html>`_
-
-CuPy also includes the following features for performance:
-
-- User-defined elementwise CUDA kernels
-- User-defined reduction CUDA kernels
-- Just-in-time compiler converting Python functions to CUDA kernels
-- Fusing CUDA kernels to optimize user-defined calculation
-- `CUB <https://github.com/NVIDIA/cub>`_/`cuTENSOR <https://developer.nvidia.com/cutensor>`_ backends for reduction and other routines
-- Customizable memory allocator and memory pool
-- `cuDNN <https://developer.nvidia.com/cudnn>`_ utilities
-- Full coverage of `NCCL <https://developer.nvidia.com/nccl>`_ APIs
-
-CuPy uses on-the-fly kernel synthesis: when a kernel call is required, it
-compiles a kernel code optimized for the shapes and dtypes of given arguments,
-sends it to the GPU device, and executes the kernel. The compiled code is
-cached to ``$(HOME)/.cupy/kernel_cache`` directory (this cache path can be
-overwritten by setting the ``CUPY_CACHE_DIR`` environment variable). It may
-make things slower at the first kernel call, though this slow down will be
-resolved at the second execution. CuPy also caches the kernel code sent to GPU
-device within the process, which reduces the kernel transfer time on further
-calls.
+`CuPy <https://github.com/cupy/cupy>`__ is a NumPy/SciPy-compatible array library for GPU-accelerated computing with Python.
+CuPy acts as a drop-in replacement to run existing NumPy/SciPy code on `NVIDIA CUDA <https://developer.nvidia.com/cuda-toolkit>`__ or `AMD ROCm <https://www.amd.com/en/graphics/servers-solutions-rocm>`__ platforms.
+
+CuPy provides a ``ndarray``, sparse matrices, and the associated routines for GPU devices, all having the same API as NumPy and SciPy:
+
+* **N-dimensional array**: :class:`cupy.ndarray`
+
+  * Data types (dtypes): boolean (``bool_``), integer (``int8``, ``int16``, ``int32``, ``int64``, ``uint8``, ``uint16``, ``uint32``, ``uint64``), float (``float16``, ``float32``, ``float64``), and complex (``complex64``, ``complex128``)
+  * Supports the semantics identical to :class:`numpy.ndarray`, including basic / advanced indexing and broadcasting
+
+* **Sparse matrices**: :mod:`cupyx.scipy.sparse`
```

Comment:
Maybe we should hyperlink to the base class to be consistent in style?
```suggestion
* **Sparse matrices**: :class:`cupyx.scipy.sparse.spmatrix`
```

Reply:
I rather changed `cupy.ndarray` link to `reference/ndarray` instead, as docs for `ndarray` and `spmatrix` are not very interesting 😅